apps/aecore/src/aec_dev_reward.erl

```diff
@@ -0,0 +1,135 @@
+-module(aec_dev_reward).
+
+-export([ensure_env/0]). % setup hook
+
+-export([enabled/0,
+         activated/1,
+         beneficiaries/0,
+         allocated_shares/0,
+         total_shares/0,
+         split/3]).
+
+-ifdef(TEST).
+-export([split_int/5]).
+-endif.
+
+-include_lib("aecontract/include/hard_forks.hrl").
+
+
+-define(ENABLED, false).
+-define(TOTAL_SHARES, 1000). % 100 shares == 10% of the reward
+%%% for: "ak_2KAcA2Pp1nrR8Wkt3FtCkReGzAi8vJ9Snxa4PcmrthVx8AhPe8:100"
+-define(BENEFICIARIES, [{<<172,241,128,85,116,104,119,143,197,105,4,192,224,207,200,138,230,84,111,38,89,33,239,21,201,183,185,209,19,60,109,136>>, 100}]).
```

Comment:
100 will be the percentage of shares? how can I validate the binary corresponds to the given account?

Reply:
> how can I validate the binary corresponds to the given account?

```
$ docker run --rm -it --entrypoint bin/aeternity --network=none aeternity/aeternity:v2.5.0 console
...
(aeternity@localhost)1> aeser_api_encoder:encode(account_pubkey, <<172,241,128,85,116,104,119,143,197,105,4,192,224,207,200,138,230,84,111,38,89,33,239,21,201,183,185,209,19,60,109,136>>).
<<"ak_2KAcA2Pp1nrR8Wkt3FtCkReGzAi8vJ9Snxa4PcmrthVx8AhPe8">>
```